app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -129,6 +136,45 @@ public void onDrawerStateChanged(int newState){
         }
 
         checkInformedConsent();
+
+        // XXX: This is probably not correct: we would like to send
+        // info to the orchestrator only when the network or any other
+        // orchestrator parameter like country code changed.
```

Comment:
The network change or other listeners havent been implemented yet. I will remove this code in 1.1.6